include/gz/rendering/Light.hh

```diff
@@ -26,6 +26,24 @@ namespace gz
   namespace rendering
   {
     inline namespace GZ_RENDERING_VERSION_NAMESPACE {
+
+    /// \enum LightType
+    /// \brief Enum for Light types.
+    enum class GZ_RENDERING_VISIBLE LightType
+    {
+      /// \brief No light type specified
+      EMPTY = 0,
```

Comment:
was there an issue with what you had before, `LT_EMPTY`?

Reply:
Changing `LightType` to enum class ensures any usage of it is like `rendering::LightType::DIRECTIONAL` (or other light type values) as opposed to `rendering::LT_DIRECTIONAL`, so no need for `LT_` in the naming anymore.